spec/label_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec'
+require_relative '../item'
+require_relative '../Book/label'
+
+RSpec.describe Label do
+  describe '#item_add' do
+    it 'adds the item to the label' do
+      label = Label.new('My Label', 'red')
+      item = double('Item')
+      expect(item).to receive(:label=).with(label)
+      label.item_add(item)
+      expect(label.items).to include(item)
+    end
+  end
```

Comment:
- [ ] @team great work adding tests for label class, however its still missing test for initialize and to_json methods, kindly consider adding them.